src/dapla_metadata/datasets/user_info.py

```diff
@@ -86,3 +86,16 @@ def get_user_info_for_current_platform() -> UserInfo:
             "Was not possible to retrieve user information! Some fields may not be set.",
         )
         return UnknownUserInfo()
+
+
+def get_owner() -> str:
+    """Returns the owner read from the GROUP_CONTEXT environment variable."""
+    if group := config.get_group_context():
+        return parse_team_name(group)
+    raise TypeError
```

Comment:
```suggestion
    raise OSError("DAPLA_GROUP_CONTEXT environment variable not found")
```

I don't think `TypeError` is suitable in this case ref: 
- https://docs.python.org/3/library/exceptions.html#OSError
- https://docs.python.org/3/library/exceptions.html#TypeError